main.js

```diff
@@ -0,0 +1,72 @@
+//1.
+function minNumber(parametr_1, parametr_2, parametr_3, parametr_4) {
+  const min = Math.min(parametr_1, parametr_2, parametr_3, parametr_4);
+  if (!isNaN(min)) return min;
+  else {
+    console.log("извините не могу посчитать");
+  }
+}
+
+const searchNumber = minNumber(-1, 3, 4, 10);
+
+//2.
+
+function NOD(num__a, num__b) {
+  if (num__b === 0) {
+    return num__a;
+  }
+  return NOD(num__b, num__a % num__b);
+}
+
+NOD(120, 50);
```

Comment:
What if 
```suggestion
NOD(120);
```
our program will stack 😢